Data/HashMap/Strict.hs

```diff
@@ -249,6 +254,54 @@ alter f k m =
     Just v  -> insert k v m
 {-# INLINABLE alter #-}
 
+-- | /O(log n)/  The expression (@'alterF' f k map@) alters the value @x@ at
+-- @k@, or absence thereof. @alterF@ can be used to insert, delete, or update
+-- a value in a map.
+--
+-- Note: 'alterF' is a flipped version of the 'at' combinator from
+-- 'Control.Lens.At'.
+--
```

Comment:
I think you need a URL; we don't depend on `lens` here!

Reply:
Fixed. This is also broken in `containers`.